### PR TITLE
Minor update to install certificates documentation

### PR DESCRIPTION
**Pull request by shubham1172**
Signed-off-by: Shubham Sharma <shubhash@microsoft.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Move the info outside codetabs as it's applicable for both self-hosted and k8s.

### CODE REVIEW

File: daprdocs/content/en/operations/configuration/install-certificates.md [hunk 1]
```diff
@@ -84,12 +84,12 @@ spec:
 When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.
 {{% /alert %}}
 
+{{% /codetab %}}
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/configuration/install-certificates.md:87]: Just noticed this: alert shortcode (lines 83-85) doesn't work within codetab shortcode, so we need to change to:

**Note**: When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.

**Review by msfussell — APPROVED** [review]
LGTM

File: daprdocs/content/en/operations/configuration/install-certificates.md (hunk 1 above)
Comment (shubham1172) [reply daprdocs/content/en/operations/configuration/install-certificates.md:87]: https://v1-8.docs.dapr.io/operations/configuration/install-certificates/ it is working here, right? @hhunter-ms

Comment (hhunter-ms) [reply daprdocs/content/en/operations/configuration/install-certificates.md:87]: hm sort of? it puts the note into a code block kind of window. I can update later.

Comment (shubham1172) [reply daprdocs/content/en/operations/configuration/install-certificates.md:87]: Ah, I see what you mean. OK, thank you!

Comment (shubham1172) [reply daprdocs/content/en/operations/configuration/install-certificates.md:87]: Hey @hhunter-ms, this is fixed in my recent PR https://github.com/dapr/docs/pull/2628, please let me know if anything else is to be changed. Thanks!

**Pull request merged**